pkg/doc/vc/status/csl/csl.go

```diff
@@ -190,7 +190,7 @@ func (c *CredentialStatusManager) createStatusCredential(v *verifiable.Credentia
 		return nil, fmt.Errorf("create credential marshalling failed: %s", err.Error())
 	}
 
-	validatedStatusCred, _, err := verifiable.NewCredential(cred)
+	validatedStatusCred, err := verifiable.ParseCredential(cred)
```

Comment:
What is the reason for moving from 'NewCredential' to 'ParseCredential'?

Reply:
no sure, these are sync up changes with Aries framework
> What is the reason for moving from 'NewCredential' to 'ParseCredential'?
@kdimak Can you plz check this

Reply:
this was refactored in aries and is correct.